src/facade/dragonfly_connection.cc

```diff
@@ -698,6 +698,10 @@ auto Connection::FromArgs(RespVec args, mi_heap_t* heap) -> Request* {
   return req;
 }
 
+void Connection::RequestShutdown() {
```

Comment:
I understand why you introduced this function (btw, you are welcome to shoot a pr to helio and change the visibility).

However, i think word Request is misleading. Maybe ShutdownSelf?

Reply:
I thought of just overriding Shutdown in the child - but there is a struct called the same. Request like the verb, not the noun
But lets use your suggestion

Reply:
yeah, I misunderstood request.